stix_shifter_modules/demo_template/stix_translation/json/from_stix_map.json

```diff
@@ -0,0 +1,53 @@
+{
+  "ipv4-addr": {
+    "fields": {
+      "value": ["SourceIpV4-dialect1", "DestinationIpV4-dialect1"]
```

Comment:
remove instances of `dialect1` since the demo template will only have the default.